wgsl/index.bs

```diff
@@ -11737,8 +10999,861 @@ When calling a built-in function, all arguments to the function are evaluated
+  <tr><td>Parameterization
+      <td>`T` is a [=type/concrete=] [=numeric scalar=] or [=type/concrete=] [=numeric vector=]
+  <tr><td>Description
+      <td>Identity transform.<br>
+      [=Component-wise=] when `T` is a [=vector=].<br>
+      The result is `e`.
+</table>
+<table class='data builtin'>
+  <tr><td style="width:10%">Overload
+      <td><xmp highlight=rust>@const @must_use fn bitcast<T>(e : S) -> T</xmp>
+  <tr><td>Parameterization
+      <td>`S` is i32, u32, or f32<br>
+      `T` is not `S` and is i32, u32, or f32
+  <tr><td>Description
+      <td>Reinterpretation of bits as `T`.<br>
+      The result is the reintepretation of bits in `e` as a `T` value.
+</table>
+<table class='data builtin'>
+  <tr><td style="width:10%">Overload
+      <td><xmp highlight=rust>@const @must_use fn bitcast<vecN<T>>(e : vecN<S>) -> T</xmp>
+  <tr><td>Parameterization
+      <td>`S` is i32, u32, or f32<br>
+      `T` is not `S` and is i32, u32, or f32
+  <tr><td>Description
+      <td>[=Component-wise=] reinterpretation of bits as `T`.<br>
+      The result is the reintepretation of bits in `e` as a `vecN<T>` value.
+</table>
+<table class='data builtin'>
+  <tr><td style="width:10%">Overload
+      <td><xmp highlight=rust>@const @must_use fn bitcast<T>(e : vec2<16>) -> T</xmp>
+  <tr><td>Parameterization
+      <td>`T` is i32, u32, or f32
+  <tr><td>Description
+      <td>[=Component-wise=] reinterpretation of bits as `T`.<br>
+      The result is the reintepretation of the 32 bits in `e` as a `T` value, following the internal layout rules.
+</table>
+<table class='data builtin'>
+  <tr><td style="width:10%">Overload
+      <td><xmp highlight=rust>@const @must_use fn bitcast<T>(e : vec4<f16>) -> T</xmp>
+  <tr><td>Parameterization
+      <td>`T` is i32, u32, or f32
+  <tr><td>Description
+      <td>[=Component-wise=] reinterpretation of bits as `T`.<br>
+      The result is the reintepretation of the 64 bits in `e` as a `T` value, following the internal layout rules.
+</table>
+<table class='data builtin'>
+  <tr><td style="width:10%">Overload
+      <td><xmp highlight=rust>@const @must_use fn bitcast<vec2<f16>>(e : T) -> vec2<f16></xmp>
+  <tr><td>Parameterization
+      <td>`T` is i32, u32, or f32
+  <tr><td>Description
+      <td>[=Component-wise=] reinterpretation of bits as f16.<br>
+      The result is the reintepretation of the 32 bits in `e` as an f16 value, following the internal layout rules.
+</table>
+<table class='data builtin'>
+  <tr><td style="width:10%">Overload
+      <td><xmp highlight=rust>@const @must_use fn bitcast<vec4<f16>>(e : vec2<T>) -> vec4<f16></xmp>
+  <tr><td>Parameterization
+      <td>`T` is i32, u32, or f32
+  <tr><td>Description
+      <td>[=Component-wise=] reinterpretation of bits as `vec2<f16>`.<br>
+      The result is the reintepretation of the 64 bits in `e` as an f16 value, following the internal layout rules.
+</table>
+
+The internal layout rules are described in [[#internal-value-layout]].
```

Comment:
I'd probably move this to the top of the section.

(Yes, this is a pre-existing problem.)